benchmark/fluid/run_fluid_benchmark.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+PADDLE_TRAINING_ROLE=PSERVER PADDLE_PSERVER_PORT=7164 PADDLE_PSERVER_IPS=127.0.0.1 PADDLE_TRAINERS=2 PADDLE_CURRENT_IP=127.0.0.1 PADDLE_TRAINER_ID=0 python fluid_benchmark.py --model resnet --device GPU --update_method pserver --iterations=10000 &
```

Comment:
Seems this command would print all logs on the terminal, we can startup them as follows:
```bash
PADDLE_TRAINING_ROLE=PSERVER ... stdbuf -oL nohup python fluid_benchmark.py <args> 2>&1 > server.log &
```

And then users would check the logs in the `server.log` file.

Reply:
I think it's fine to print out some logs to give user some feedback. There aren't many outputs

```
Pass 0, batch 162, loss [2.7855887 2.973915 ]
Pass 0, batch 162, loss [3.0754983 3.2426462]
Pass 0, batch 171, loss [3.4701207 4.438573 ]
Pass 0, batch 171, loss [3.7791452 3.3191109]
```